src/components/element-editor/conditional-rendering/conditional-rendering.tsx

```diff
@@ -46,6 +51,47 @@ export const ConditionalRendering = () => {
         return options;
     };
 
+    function getTouchedAndErrorsFields() {
+        let count = 0;
+        let hasEmptyField = false;
+        const conditionalRenderingFields = getValues("conditionalRendering");
+
+        if (touchedFields?.conditionalRendering?.length > 0) {
+            touchedFields.conditionalRendering.forEach((field) => {
+                count += Object.keys(field).length;
+            });
+
+            conditionalRenderingFields?.forEach((field) => {
+                if (Object.values(field).includes("")) {
+                    hasEmptyField = true;
+                }
+            });
+
+            const hasErrors = errors?.conditionalRendering?.length > 0;
+            const incompleteFields =
+                count !== childEntryValues.length * 3 &&
+                hasEmptyField &&
+                childEntryValues?.length > 0;
+
+            return hasErrors || incompleteFields;
+        } else {
+            const hasErrors = errors?.conditionalRendering?.length !== 0;
+            const noTouchedFields =
+                touchedFields?.conditionalRendering?.length === undefined;
+            const hasChildEntries = childEntryValues?.length > 0;
+
+            return hasErrors && noTouchedFields && hasChildEntries;
+        }
```

Comment:
instead of coming up with the validation manually, you should rely on the actual Yup validation schema since that is the source of truth

```
 SchemaHelper.buildSchema(focusedElement.element.type)
            .pick(["conditionalRendering"])
            .validate(childEntryValues);
```

for touched fields, regardless whether they are touched or not, that is redundant right?